Week3/More Javascript/script.js

```diff
@@ -0,0 +1,148 @@
+//1
+
+function info(a, b, c) {
+    console.log(a + b + c)
+}
+info(1, 5, 34);
+
+//2
+
+function colorCar(z) {
+    print = `a ${z} car`
+    console.log(print);
+}
+colorCar("red");
+
+//3
+
+let obj = {
+    name: "Ahmet Husrev",
+    surname: "Semiz",
+    sex: "Male",
+};
+
+function objPrint(obj) {
+
+    console.log(obj);
+}
+objPrint(obj);
+
+//4
+
+function vehicleType(a, b) {
+    if (b === 1) {
+        console.log(`a ${a} car`) ;
+    } else if (b === 2) {
+        console.log(`a ${a} motorbike`) ;
+    }
+}
+vehicleType("blue", 2);
+
+//5 
+
+console.log(3 === 3 ? "yes" : "no");
+
+//6
+
+let motor = [0,"car", "motorbike"];
+
+function vehicle(color, value, index) {
+    if (index === 4) {
+        return `a ${color} new ${motor[value]}`
+    }else if ( index === 5){
+        return `a ${color} used ${motor[value]}`
+    }else{
+        return `invalid access`
+    }
+}
+
+console.log(vehicle("blue", 1, 5));
+
+//7 - 8
+
+let vehicles = ["car", "motobike", "caravan", "bike", "atv"]
+
+console.log(vehicles[2]);
+
+//9
+
+
+function vehicle (color, index, value){
```

Comment:
Same comment as above:) And this should use the same list as in assignment 6.

Reply:
Yes, It is same excepts vehicles array.